delatore/sources/awx_api.py

```diff
@@ -21,48 +25,62 @@ class NoSuchTemplate(Exception):
     """Template not found"""
 
 
+TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+"""Timestamp format with milliseconds"""
+
+
 class TemplateStatus(NamedTuple):
     name: str
     last_run_timestamp: str
     last_status: str
     playbook: str
```

Comment:
never used

Reply:
Used in _status_json

Reply:
TIMESTAMP_FORMAT used in `__md__` and in tests